src/js/wall/FirstUser.tsx

```diff
@@ -0,0 +1,101 @@
+import { useMutation } from "@tanstack/react-query";
+import React from "react";
+import { useForm } from "react-hook-form";
+import { InputError, InputGroup, InputLabel, InputSimple } from "../base";
+import { createFirst } from "../users/api";
+import { User } from "../users/types";
+import { WallButton, WallContainer, WallDialog, WallHeader, WallLoginContainer, WallSubheader } from "./Container";
+import { WallTitle } from "./WallTitle";
+
+type ErrorResponse = {
+    response: {
+        body: {
+            message: string;
+        };
+    };
+};
+
+type NewUser = {
+    handle: string;
+    password: string;
+    /** Whether the user will be forced to reset their password on next login */
+    forceReset: boolean;
+};
+
+type FormData = {
+    username: string;
+    password: string;
+};
+
+/**
+ * A form for creating the first instance user
+ */
+export default function FirstUser() {
+    const mutation = useMutation<User, ErrorResponse, NewUser>(createFirst, {
+        onSuccess: () => {
+            window.location.reload();
+        },
+    });
```

Comment:
You can move this into the queries component in `src/js/users/queries`. We actually made an `ErrorResponse` in `src/js/types/types` so you can just use that for the type.

Reply:
I don't see a queries component in `src/js/users/queries`, did you mean `useInfiniteFindUsers`?

Reply:
Sorry, I meant the `queries` file. You can just move the mutation in there with all the other react-query stuff and name the function `useCreateFirstUser` and have it return the mutation. The error response can just be imported from our types as well